tests/data_context/test_data_context.py

```diff
@@ -252,6 +252,14 @@ def test_data_context_expectation_suite_delete(empty_data_context):
     expectation_suites = empty_data_context.list_expectation_suite_names()
     assert len(expectation_suites) == 0
 
+def test_data_context_expectation_spchars_suite_delete(empty_data_context):
+    assert empty_data_context.create_expectation_suite(expectation_suite_name="titanic.test.create_expectation_suite")
+    expectation_suites = empty_data_context.list_expectation_suite_names()
+    assert len(expectation_suites) == 1
+    empty_data_context.delete_expectation_suite(expectation_suite_name=expectation_suites[0])
+    expectation_suites = empty_data_context.list_expectation_suite_names()
+    assert len(expectation_suites) == 0
+
```

Comment:
This appears identical to the test above. What is the test that is needed for this behavior?

Reply:
It's not identical. Specialchar is different.